themes/wim/components/wim/_block--menu-top-tasks.scss

```diff
@@ -71,7 +76,7 @@
           }
           &:after {
             position: absolute;
-            font-family: 'Glyphicons Halflings';
+            font-family: 'Glyphicons Halflings'; //@todo use fontawesome icons
```

Comment:
@Crazylord We also discussed this last time. Make use of the classes available in the fontawesome fontpack that we implemented. Extend them like I showed you. We want to be able to only override one class with a different icon font and that it changes site wide. Do not use static values.